src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -6,12 +6,20 @@ export function focusElement(selectorOrElement, options) {
     typeof selectorOrElement === 'string'
       ? document.querySelector(selectorOrElement)
       : selectorOrElement;
+  let prevTabIdx = null;
 
   if (el) {
-    if (el.tabIndex <= 0) {
+    prevTabIdx = el.getAttribute('tabindex');
```

Comment:
just curious if `el.getAttribute('tabindex')` is functionally different than `el.tabIndex`